integration_tests/cairozero_test.go

```diff
@@ -60,9 +60,9 @@ func TestCairoZeroFiles(t *testing.T) {
 
 	testFiles := append(testFiles1, testFiles2...)
 
-	testFiles1Map := make(map[string]struct{}, len(testFiles1))
```

Comment:
does it change anything to do that?

Reply:
No, its just cleaner that way. We are using map to detect whether testfile is a part of root1. And mapping to a struct does not really give away that message. But I think mapping to a bool surely does.

Reply:
we dont care of the type of the value , we just need the key

Reply:
yeah um... as I said its just to make it look cleaner.

Reply:
Oh no nothing. Its just have to be assigned something to put it in the map. It can be false also, it does not matter. I chose true just to make intuitive that this map "contains" the root1 files.

Reply:
ok so i just checked a bit:
in go, using struct{} when you just need to check for keys is better, as it does use any memory for the value, in contrary to `bool`

we should keep `struct{}{}`

Reply:
Oh, interesting. I was not aware of that.